File: pgm/ABC_Dart/ABC_434_A.dart
import 'dart:io';

void main() {
  final [int W, int B] = stdin
      .readLineSync()!
      .split(" ")
      .map(int.parse)
      .toList();
  const int one = 1;
  const int thousand = 1000;
  print((W * thousand / B + one).toInt());
  exit(0);
}

File: pgm/ABC_C#/ABC_434_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split(' ');
        const int one = 1;
        const int thousand = 1000;
        const int zero = 0;
        int W = int.Parse(arr[zero]);
        int B = int.Parse(arr[one]);
        Console.WriteLine(thousand * W / B + one);
        Environment.Exit(0);
    }
}